composer.json

```
@@ -12,13 +12,13 @@
     ],
     "require": {
         "php": ">=8.2",
-        "symfony/asset": "^6.4 || ^7.0",
-        "symfony/cache": "^6.4 || ^7.0",
-        "symfony/config": "^4.4 || ^5.3 || ^6.0 || ^7.0",
-        "symfony/dependency-injection": "^6.4 || ^7.0",
-        "symfony/finder": "^6.4 || ^7.0",
-        "symfony/framework-bundle": "^6.4 || ^7.0",
-        "symfony/http-kernel": "^6.4 || ^7.0"
+        "symfony/asset": "^6.4 || ^7.0 || ^8.9",
```

I guess this is a typo:
```suggestion
        "symfony/asset": "^6.4 || ^7.0 || ^8.0",
```